images/router/haproxy/conf/haproxy-config.template

```diff
@@ -227,6 +227,12 @@ listen stats
 
   {{ if .BindPorts -}}
 frontend public
+    {{- if ne (env "ROUTER_SYSLOG_ADDRESS") "" }}
```

Comment:
Consider defining `{{- $syslogAddress := env "ROUTER_SYSLOG_ADDRESS" }}`, `{{- $httpLogFormat := env "ROUTER_HTTP_LOG_FORMAT" }}`, and so on for the following reasons:

- Less performance hit from repeatedly calling into Go helpers.
- Less risk of typos in the environment variable's name.
- Slightly simpler and more readable `if` conditions.